### ISSUE TITLE
cannot set column size for last colulmn

### ISSUE THREAD

**Issue by adityadees** [handsontable/handsontable#9479]
### Description
I can change manualy all column except last column
this bug only appear when we use nexted header

![image](https://user-images.githubusercontent.com/37553901/167979578-518e4dd6-336a-4888-8f44-5c073e708168.png)

as you can see for the last column

this my configurations

```
autoColumnSize : false,
    stretchH: 'none',
    colWidths(index) {
        return 20;
 },
```

### Steps to reproduce

1. Use nested header
2. set column width
3. and last column didnt work

### Your environment
* Handsontable version:
* Browser name and version:
* Operating system:

**Comment by AMBudnik** [comment]
Hi @adityadees 

I just made this demo using your settings with some `nestedHeaders` defined https://jsfiddle.net/09qa17po/4/ and everything works well on the latest Chrome on Mac. 

![Zrzut ekranu 2022-05-12 o 14 11 55](https://user-images.githubusercontent.com/10757813/168072359-b1e1f7ca-e34b-49e5-8ee1-8c86aa22618a.png)

The only oddity that I spotted is that we have the resize handler for a single column on the higher levels but not for those that are merged.

Maybe it's a matter of the browser or any other settings that you have in the Handsontable configuration object.
